docs/fundamentals/tools-and-productivity.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tools and productivity in .NET
+author: IEvangelist
+description: Learn about the various tooling, software development kits, diagnostic tools, and code analyzers available to .NET developers.
+ms.author: dapine
+ms.date: 10/20/2020
+ms.topic: overview
+---
+
+# Tools and productivity in .NET
+
+In this article, you'll learn about the various tools available to .NET developers. With .NET, you have a robust software development kit (SDK), that includes a command-line interface (CLI) namely, the .NET CLI. The .NET CLI powers many of the features throughout the .NET-ready integrated development environments (IDEs). This article also provides resources to productivity capabilities, such as .NET CLI tools for diagnosing performance issues, memory leaks, high CPU, deadlocks, and as well as tooling support for code analysis.
+
+## Tools
+
+Many tools are available to .NET developers, such as the .NET SDK, .NET CLI, IDEs, and so on.
+
+### .NET SDK
+
+You can download [.NET SDK](https://dotnet.microsoft.com/download) or .NET Runtime for Windows, Linux, macOS, or Docker. For more information, see [.NET SDK overview](../core/sdk.md).
+
+### .NET CLI
+
+The .NET CLI is a cross-platform toolchain for developing, building, running, and publishing .NET applications. The .NET CLI is included with the .NET SDK. For more information, see [.NET CLI overview](../core/tools/index.md).
+
+### IDEs
+
+The .NET SDK powers language services, that exposes .NET developer functionality in [Visual Studio Code](https://code.visualstudio.com). As part of the [Visual Studio](https://visualstudio.microsoft.com/vs) family, [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac) enables macOS developers to write .NET applications.
```

Comment:
"the SDK powers" is vague language that doesn't help explain what IDEs are available and their respective advantages. A bulleted list or set  of h3s here could present the IDE choices and how you might choose among them.